bin/pycbc_live

```diff
@@ -108,6 +108,8 @@ class LiveEventManager(object):
         # Keep track of which events have been uploaded
         self.last_few_coincs_uploaded = []
 
+        # Give this a nominal value, it is requried but not used
+        self.fu_cores = None
```

Comment:
Typo: requried -> required. Also, there should probably be a `if self.fu_cores is not None` condition down below in `setup_optimize_snr` where `fu_cores` is used, otherwise I do not think the resulting command (`--cores None`) would be valid.